paper/ollie-optimization.tex

```diff
@@ -156,7 +156,7 @@ \section{Introduction}\label{s_intro}
 distinct phases which are shown in figure \ref{fig:ollie_steps}.
 
 
-\begin{figure*}
+\begin{figure*}[!t]
```

Comment:
It is best to not specify locations of figures, just let them land where they do. Then when the paper is done, if you absolutely want to move one, do it then.